src/generated/event-names.ts

```diff
@@ -1,381 +1,6 @@
-    | "sponsorship.pending_cancellation"
-    | "sponsorship.pending_tier_change"
-    | "sponsorship.tier_changed";
-
-  type StarEvent = "star" | "star.created" | "star.deleted";
-
-  type StatusEvent = "status";
-
-  type TeamEvent =
-    | "team"
-    | "team.added_to_repository"
-    | "team.created"
-    | "team.deleted"
-    | "team.edited"
-    | "team.removed_from_repository";
-
-  type TeamAddEvent = "team_add";
-
-  type WatchEvent = "watch" | "watch.started";
-
-  type WorkflowDispatchEvent = "workflow_dispatch";
-
-  type WorkflowRunEvent = "workflow_run" | "workflow_run.action";
-
-  type StringNames =
-    | "check_run"
-    | "check_suite"
-    | "code_scanning_alert"
-    | "commit_comment"
-    | "content_reference"
-    | "create"
-    | "delete"
-    | "deploy_key"
-    | "deployment"
-    | "deployment_status"
-    | "fork"
-    | "github_app_authorization"
-    | "gollum"
-    | "installation"
-    | "installation_repositories"
-    | "issue_comment"
-    | "issues"
-    | "label"
-    | "marketplace_purchase"
-    | "member"
-    | "membership"
-    | "meta"
-    | "milestone"
-    | "organization"
-    | "org_block"
-    | "package"
-    | "page_build"
-    | "ping"
-    | "project_card"
-    | "project_column"
-    | "project"
-    | "public"
-    | "pull_request"
-    | "pull_request_review"
-    | "pull_request_review_comment"
-    | "push"
-    | "release"
-    | "repository_dispatch"
-    | "repository"
-    | "repository_import"
-    | "repository_vulnerability_alert"
-    | "security_advisory"
-    | "sponsorship"
-    | "star"
-    | "status"
-    | "team"
-    | "team_add"
-    | "watch"
-    | "workflow_dispatch"
-    | "workflow_run";
-
-  type All =
-    | ErrorEvent
-    | WildcardEvent
-    | CheckRunEvent
-    | CheckSuiteEvent
-    | CodeScanningAlertEvent
-    | CommitCommentEvent
-    | ContentReferenceEvent
-    | CreateEvent
-    | DeleteEvent
-    | DeployKeyEvent
-    | DeploymentEvent
-    | DeploymentStatusEvent
-    | ForkEvent
-    | GithubAppAuthorizationEvent
-    | GollumEvent
-    | InstallationEvent
-    | InstallationRepositoriesEvent
-    | IssueCommentEvent
-    | IssuesEvent
-    | LabelEvent
-    | MarketplacePurchaseEvent
-    | MemberEvent
-    | MembershipEvent
-    | MetaEvent
-    | MilestoneEvent
-    | OrganizationEvent
-    | OrgBlockEvent
-    | PackageEvent
-    | PageBuildEvent
-    | PingEvent
-    | ProjectCardEvent
-    | ProjectColumnEvent
-    | ProjectEvent
-    | PublicEvent
-    | PullRequestEvent
-    | PullRequestReviewEvent
-    | PullRequestReviewCommentEvent
-    | PushEvent
-    | ReleaseEvent
-    | RepositoryDispatchEvent
-    | RepositoryEvent
-    | RepositoryImportEvent
-    | RepositoryVulnerabilityAlertEvent
-    | SecurityAdvisoryEvent
-    | SponsorshipEvent
-    | StarEvent
-    | StatusEvent
-    | TeamEvent
-    | TeamAddEvent
-    | WatchEvent
-    | WorkflowDispatchEvent
-    | WorkflowRunEvent;
+  type All = keyof EventTypesPayload;
 }
```

Comment:
can we get rid of this file altogether?

Reply:
Done! I only did not remove it as it was exported from entrypoint. Will this be a breaking change for the users?